assets/app/lib/settings.rb

```diff
@@ -5,6 +5,15 @@
 module Lib
   module Settings
     DARK = `window.matchMedia('(prefers-color-scheme: dark)').matches`.freeze
+    # http://mkweb.bcgsc.ca/colorblind/ 12 color palette
+    ROUTE_COLORS = %i[#A40122 #008DF9 #00FCCF #FF5AAF].freeze
+
+    routes = {}
+    ROUTE_COLORS.each_with_index do |color, n|
+      routes["r#{n}_color"] = color
```

Comment:
since you use these keys later on, i think we should make a helper function to get them so if we decide to change it it's not too error prone